### ISSUE TITLE
### SUGGESTED FEATURES: Congrats message

### ISSUE THREAD

**Issue by YuriDevAT** [YurisCodingClub/html-tag-quiz#30]
### SUGGESTED FEATURES

2. A congrats message if they entered a certain number of tags already (let’s say 50, 100 ..) or how many left until a complete list
3. Or, a congrats message if they entered all the tags that start with a certain letter (Congrats, you have now entered all the a-letter tags!)

_Originally posted by @CorinaMurg in https://github.com/YurisCodingClub/html-tag-quiz/discussions/20#discussioncomment-7233911_

**Comment by CorinaMurg** [comment]
I am working on the alphabetical sorting of the list, and one idea is for each group of tags to be a bit on its own. Visually it would be easier to look up the group starting with a certain letter. We could also mark (and let the screen reader announce) the completion of a group. And the ellipsis (...) could indicate that a group is not yet completed.
A very simplistic design to illustrate this (it was done in Excel!):
![image](https://github.com/YurisCodingClub/html-tag-quiz/assets/115652409/f3995133-2c29-472a-b26e-eec8229d761c)

This could take care of letting users know when they completed one group.
We don't need to have the heading with the group's letter, but at least a gap between each group.

And then of course, announce 50% and 100% completion.  
We could also add a countdown when there are only 10 or just 3 left to submit.

**Comment by YuriDevAT** [comment]
Great idea! This makes it easier to follow the progress.

**Comment by CorinaMurg** [comment]
@YuriDevAT 
Do you think this is ready to move forward? By this I mean define the issue more clearly and implement it? I think I was waiting for us to have a UI design ready, but now I realize that the JS parts of this issue could be implemented regardless of the design.

**Comment by YuriDevAT** [comment]
Yeah of course. The content of the message can be separated from the design, I would say. This issue is more about deciding how many messages will there be and what they are going say.

**Comment by CorinaMurg** [comment]
@YuriDevAT 
Right now we have an ariaMessage that updates with each new added tag, so a screen reader user will hear "23 added" etc.
This changes to a congrats message at 50% and 100% completion. 

Do we want this info displayed also on screen? Not sure if it's necessary since we have the progress bar, but it would be more specific than the progress bar, which some users might prefer. 
![image](https://github.com/YurisCodingClub/html-tag-quiz/assets/115652409/620d0a59-a561-4aa5-9096-90dd7aff9203)

![image](https://github.com/YurisCodingClub/html-tag-quiz/assets/115652409/1e58beee-9f78-43f9-87be-09379ed7982c)

Not sure what would be best.

**Comment by YuriDevAT** [comment]
Yes, my suggestion would be as I added it into the design, having the progress showing as a number as well.
<img width="259" alt="Progressbar showing current number of added tags." src="https://github.com/YurisCodingClub/html-tag-quiz/assets/54622834/dcfecffb-1787-4bfa-bac1-b7a92e3ca257">
